docs/pages/enroll-resources/server-access/guides/encrypted-session-recordings/encrypted-session-recordings.mdx

```diff
@@ -60,6 +60,11 @@ are the default, but other backends can also be configured:
 - [Google Cloud KMS](../../../../zero-trust-access/deploy-a-cluster/gcp-kms.mdx)
 - [HSM](../../../../zero-trust-access/deploy-a-cluster/hsm.mdx)
 
+If your cluster is already configured to use one of these backends, you may need
```

Comment:
Since there are only three backends listed here, I think it would make sense to specify whether each one requires adding permissions. Otherwise, the use of "you may need" here makes things uncertain for the reader.